src/js/view/controls/controls.js

```diff
@@ -1,31 +1,361 @@
+                            }
+                        }
+                        break;
+                    case 77: // m-key
+                        api.setMute();
+                        break;
+                    case 70: // f-key
+                        api.setFullscreen();
+                        break;
+                    default:
+                        if (evt.keyCode >= 48 && evt.keyCode <= 59) {
+                            // if 0-9 number key, move to n/10 of the percentage of the video
+                            const number = evt.keyCode - 48;
+                            const newSeek = (number / 10) * model.get('duration');
+                            api.seek(newSeek, reasonInteraction());
+                        }
+                }
+
+                if (/13|32|37|38|39|40/.test(evt.keyCode)) {
+                    // Prevent keypresses from scrolling the screen
+                    evt.preventDefault();
+                    return false;
+                }
+            };
+            this.playerContainer.addEventListener('keydown', handleKeydown);
+            this.keydownCallback = handleKeydown;
+
+            // Show controls when enabled
+            this.userActive();
 
-        enable() {
             this.playerContainer.appendChild(this.element);
         }
 
         disable() {
-            this.playerContainer.removeChild(this.element);
+            this.off();
+            clearTimeout(this.activeTimeout);
+
+            if (this.element.parentNode) {
+                this.playerContainer.removeChild(this.element);
+            }
+            if (this.controlbar) {
+                this.removeActiveListeners(this.controlbar.element());
+            }
+            if (this.rightClickMenu) {
+                this.rightClickMenu.destroy();
+                this.rightClickMenu = null;
+            }
+
+            if (this.keydownCallback) {
+                this.playerContainer.removeEventListener('keydown', this.keydownCallback);
+                this.keydownCallback = null;
+            }
         }
 
         getElement() {
             return this.element;
         }
+
+        resize(model, breakPoint) {
+            const audioMode = isAudioMode(model);
+
+            // Set timeslider flags
+            const smallPlayer = breakPoint < 2;
+            const timeSliderAboveConfig = model.get('timeSliderAbove');
+            const timeSliderAbove = !audioMode &&
+                (timeSliderAboveConfig !== false) && (timeSliderAboveConfig || smallPlayer);
+            utils.toggleClass(this.playerContainer, 'jw-flag-small-player', smallPlayer);
+            utils.toggleClass(this.playerContainer, 'jw-flag-audio-player', audioMode);
+            utils.toggleClass(this.playerContainer, 'jw-flag-time-slider-above', timeSliderAbove);
+
+            model.set('audioMode', audioMode);
+        }
+
+        unmuteAutoplay(api, model) {
+            const autostartSucceeded = !model.get('autostartFailed');
```

Comment:
@egreaves implemented this. It is not set by default, and later set to `undefined`. search.